### ISSUE TITLE
Unsubscribing to notifications on your own post doesn't work

### ISSUE THREAD

**Issue by michael-tharrington** [thepracticaldev/dev.to#8745]
**Describe the bug**

When a user tries to unsubscribe from the notifications on a single post they've written, they still receive notifications on that post. 

**To Reproduce**

1. Navigate to a post you've written,
2. Scroll down to the comments/discussion section
3. Click the unsubscribe button (image below) 
4. See the success message that you've unsubscribed. (It briefly appears in the bottom left. You should also see that the unsubscribe button has been replaced with a subscribe button.)
5. Use a different DEV account to comment on your post OR ask another DEV user to comment on your post.
6. Look at https://dev.to/notifications and you'll see a notification for that comment.  

**Expected behavior**

If a user unsubscribes from comments on their own post, they should stop receiving comment notifications here - https://dev.to/notifications.

**Screenshots**

<img width="583" alt="Image 2020-06-17 at 2 52 10 PM" src="https://user-images.githubusercontent.com/16007075/84937697-2d59a500-b0aa-11ea-8c70-729bcd00d288.png">

**Additional context**

Users can also control which notifications they receive from here in their settings - https://dev.to/settings/notifications.

If they've unsubscribed from the comments on their own post, then I think that should override a few of the scenarios they may have checked in their notification settings. 

For instance, under email notification settings, if they have checked "Send me an email when someone replies to me in a comment thread" I think that they shouldn't receive this email if they have opted to not receive notifications on their article and someone comments on a comment they made in the discussion section of that article. Similarly, under mobile notifications settings, if they have checked "Notify me when someone replies to me in a comment thread" I think that they shouldn't receive this notification if they have opted to not receive notifications on their article and someone comments on a comment they made in the discussion section of that article.

**Comment by citizen428** [comment]
To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by Oghenebrume50** [comment]
Hi, @michael-tharrington @citizen428 I tried this in both the production site and in my local setup, but I could not reproduce it, @michael-tharrington could you please check again, or maybe someone else could check just to make sure this hasn't been resolved already

**Comment by michael-tharrington** [comment]
Hey @Oghenebrume50! Well dang, I just tried this on DEV actually with my [test account](https://dev.to/michaelctestington111) (yeah, that's not a great practice...) and also could not repro it. It was originally reported to me by another user, but I was able to confirm it myself the first time around.

I'm going to close this as it appears to be resolved, but we can open it back up if any weirdness starts to occur.

Thanks for looking into it and sorry for leading you to a dead end @Oghenebrume50 !
